src/Faker/Provider/et_EE/Person.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Faker\Provider\et_EE;
+
+/**
+ * Class Person
+ * @package Faker\Provider\et_EE
+ * @author David Gegelija <code@imdavid.xyz>
+ */
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @link https://news.err.ee/114745/most-popular-baby-names-of-2014
+     */
+    protected static $firstNameMale = array(
+        'Rasmus', 'Artjom', 'Robin', 'Martin', 'Oliver', 'Romet', 'Sebastian', 'Robert',
+        'Artur', 'Maksim', 'Markus', 'Marten', 'Karl', 'Kristofer', 'Oskar', 'Daniel',
+        'Hugo', 'Henri', 'Mark', 'Nikita', 'Kirill', 'Sander', 'Kevin', 'Aleksandr', 'Daniil'
+    );
+
+    /**
+     * @link https://news.err.ee/114745/most-popular-baby-names-of-2014
+     */
+    protected static $firstNameFemale = array(
+        'Sofia', 'Eliise', 'Maria', 'Mia', 'Lisandra', 'Mirtel', 'Sandra', 'Emma', 'Laura',
+        'Darja', 'Arina', 'Milana', 'Alisa', 'Anastasia', 'Lenna', 'Liisa', 'Anna', 'Viktoria',
+        'Elisabeth', 'Polina', 'Marta', 'Aleksandra', 'Marleen', 'Hanna', 'Nora'
+    );
+
+    /**
+     * @link https://en.wikipedia.org/wiki/Category:Estonian-language_surnames
```

Comment:
From what I understand, it's based on the articles on Wikipedia about Estonian people. How is that representative?